Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/debugoptions/ui/DebugOptionsFragmentViewModel.kt

```diff
@@ -52,7 +55,9 @@ class DebugOptionsFragmentViewModel @AssistedInject constructor(
     }
 
     private fun cleanCachedData() = appScope.launch {
-        environmentSunset.reset()
+        if (!eol.isEol.first()) {
```

Comment:
Better add this check before calling cleanCachedData() or rename cleanCachedData() to cleanCachedDataIFEol()

Reply:
Ok, I'll rename it. I added it here because we were already doing `.launch{}` so I could read the value from the flow.